config/routes.rb

```diff
@@ -1,22 +1,27 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-  get '/merchants/:id/items', to: 'merchant_items#index'
-  get '/merchants/:merchant_id/items/:item_id', to: 'merchant_items#show'
-  get '/merchants/:merchant_id/items/:item_id/edit', to: 'merchant_items#edit'
-  patch '/merchants/:merchant_id/items/:item_id', to: 'merchant_items#update'
-
+  # get '/merchants/:id/items', to: 'merchant_items#index'
+  # get '/merchants/:merchant_id/items/:item_id', to: 'merchant_items#show'
+  # get '/merchants/:merchant_id/items/:item_id/edit', to: 'merchant_items#edit'
+  # patch '/merchants/:merchant_id/items/:item_id', to: 'merchant_items#update'
+  resources :merchants do
```

Comment:
should we throw an only: [:index] here to limit the number of merchants routes created?